notebooks_tsqr/NightLog.ipynb

```diff
@@ -19,7 +19,8 @@
     "# Times Square replaces this cell with the user's parameters.\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    instrument = 'lsstcomcamsim' # Which instrument to query for data (consdb)
</pre><p>NightLog should report on ALL instruments.  It shouldn't require the user to pick just one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_logging_and_reporting/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Refactored to accommodate this